### PR TITLE
Upgrade to Kibana 7.7.0

### PR DESCRIPTION
**Pull request by ohltyler**
*Issue #, if available:* #145 

*Description of changes:*

This PR upgrades the plugin to be compatible with Kibana 7.7.0 and Open Distro for Elasticsearch 1.8.0.

Changes include:
- Updates unit test workflow to test in Kibana 7.7.0 repository
- Bumps plugin version to 1.8.0.0
- Changes health colors to be more readable in Kibana light mode
- Updates dependencies in yarn.lock
- Adds Open Distro for Elasticsearch 1.8.0 release notes (will need to add additional PRs that are included in this release later)

Local unit tests:
Screen Shot 2020-05-19 at 2 52 24 PM

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: release-notes/opendistro-for-elasticsearch.anomaly-detection-kibana.release-notes-1.8.0.0.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+## Open Distro for Elasticsearch 1.7.0 Release Notes
+Compatible with Kibana 7.7.0 and Open Distro for Elasticsearch 1.8.0.
+
+## Enhancements
```
Comment (ylwu-amzn) [inline release-notes/opendistro-for-elasticsearch.anomaly-detection-kibana.release-notes-1.8.0.0.md:4]: Will we add other changes to 1.8 ?

Comment (ohltyler) [reply release-notes/opendistro-for-elasticsearch.anomaly-detection-kibana.release-notes-1.8.0.0.md:4]: yeah, this is just preliminary. will need to add additional PRs once we have all changes we want to include for this release

File: public/pages/utils/constants.ts [hunk 2]
```diff
@@ -16,13 +16,18 @@
 import { SORT_DIRECTION } from '../../../server/utils/constants';
 import { DETECTOR_STATE } from '../../utils/constants';
 
+export const newSubduedColor = '#98A2B3';
```
Comment (yizheliu-amazon) [inline public/pages/utils/constants.ts:19]: `CustomizedSubduedColor` may be a better name. Same for below new colors.

Comment (ohltyler) [reply public/pages/utils/constants.ts:19]: Agreed, will change

**Comment by yizheliu-amazon** [comment]
LGTM. Thanks for the change.

**Pull request merged**